index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
```

Comment:
- [ ] As requested by the previous reviewers. Please kindly use inline comments to give your review. This is an example of an inline comment. In order to give inline comment, click on the `files changed`, and scroll to the particular line of code you want to comment on. Click on that line to add a comment. After you are done reviewing, click on the `Finish your review` button, add a short summary and select 'comment' or 'require changes'. Please you can check out more about the steps in [this link](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request).

- [Optional] Please phrase the comments politely, as suggestions instead of orders. For example, This is a polite suggestion: *"I think it would be better to use semantic tags"*, this comment is an order: *"Use semantic tags"*

- [Optional] Please consider using a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖😁.

- [Optional] Please kindly consider including an explanation, a "why", for each requested change. For example, This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*, This request lacks a "why": *Why don't you use an article tag here instead of a div?*